elasticsearch/templates/statefulset.yaml

```diff
@@ -111,6 +111,14 @@ spec:
           configMap:
             name: {{ template "uname" . }}-config
         {{- end }}
+{{ if .Values.keystore }}
```

Comment:
Did you want to indent this and use `{{- if ...` ? Not sure if we're concerned with indentation style.

Reply:
Fixed! It would be nice to add a "white space detection" check in the pull request testing and do a one off PR to get things back in line. Right now its a bit of a mess. It's a shame that `helm lint --strict` doesn't already cover this.